src/sonic-config-engine/tests/test_cfggen.py

```diff
@@ -50,9 +50,11 @@ def tearDown(self):
         except OSError:
             pass
 
-    def run_script(self, argument, check_stderr=False, verbose=False):
+    def run_script(self, argument, check_stderr=False, verbose=False, skip_yang_validation=False):
         print('\n    Running sonic-cfggen ' + ' '.join(argument))
-        self.assertTrue(self.yang.validate(argument))
+        if not skip_yang_validation:
```

Comment:
Discussed in Yang community (1/19) and proposal is to add the Yang model and remove the skip validation

Reply:
This is addressed

Reply:
Could you share the link addressed this issue?